.circleci/config.yml

```diff
@@ -48,24 +47,27 @@ jobs:
           command: |
             ./tmp/cc-test-reporter after-build -t gocov --prefix github.com/nuts-foundation/nuts-node
 
-  e2e-tests:
-    machine:
-      image: ubuntu-2004:202104-01
-
-    steps:
-      - checkout
-      - run:
-          name: Build image
-          command: |
-            docker build -t nuts-node-ci/${CIRCLE_SHA1} .
-      - run:
-          name: Clone E2E tests repo
-          command: |
-            git clone https://github.com/nuts-foundation/nuts-go-e2e-test.git && \
-              cd nuts-go-e2e-test && \
-              find . -type f -name "docker-compose.yml" | xargs -I{} sed "s/nutsfoundation\/nuts-node:master/nuts-node-ci\/${CIRCLE_SHA1}/g" {}
-      - run:
-          name: Run E2E tests
-          command: |
-            cd nuts-go-e2e-test && \
-              docker run --privileged -t -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd):/workspace -v earthly-tmp:/tmp/earthly:rw earthly/earthly:latest -P +all
+#  e2e-tests:
```

Comment:
just remove the commented stuff. Will be in Github history if you need it.

Reply:
Oops, removed!